docs/configuration.html

```diff
@@ -70,7 +70,7 @@
 
 <h3><a id="consumerconfigs" href="#consumerconfigs">3.3 Consumer Configs</a></h3>
 
-We introduce both the old 0.8 consumer configs and the new consumer configs respectively below.
+We introduce both the old 0.8 consumer configs and the Java consumer configs respectively below.
```

Comment:
Might be just me, but it reads a little awkwardly to mention the "0.8 consumer configs" contrasted with the "Java consumer configs." Maybe no need to mention versions? Also maybe we should swap the order?
```
We introduce both the Java consumer configs and the older Scala consumer configs respectively below.
```